cpp/models/abm/location.h

```diff
@@ -137,33 +131,52 @@ class Location
         return m_index;
     }
 
+    /**
+     * @brief Compute the transmission factor for contact transmission of the virus in a Cell.
+     * @param[in] cell_index Cell index of the Cell.
+     * @param[in] virus VirusVariant of interest.
+     * @param[in] age_receiver AgeGroup of the receiving Person.
+     * @param[in] age_transmitter AgeGroup of the transmitting Person.
+     * @returns Amount of average infections with the virus from the AgeGroup of the transmitter per day.
+    */
+    ScalarType transmission_contacts_per_day(uint32_t cell_index, VirusVariant virus, AgeGroup age_receiver,
+                                             AgeGroup age_transmitter) const;
+
+    /**
+     * @brief Compute the transmission factor for a aerosol transmission of the virus in a Cell.
+     * @param[in] cell_index Cell index of the Cell.
+     * @param[in] virus VirusVariant of interest.
+     * @returns Amount of average infections with the virus per day.
+    */
+    ScalarType transmission_air_per_day(uint32_t cell_index, VirusVariant virus) const;
+
     /** 
      * @brief A person interacts with the population at this Location and may become infected.
      * @param person The Person that interacts with the population.
      * @param dt Length of the current simulation time step.
      * @param global_params Global infection parameters.
      */
-    void interact(Person& person, TimePoint t, TimeSpan dt, const GlobalInfectionParameters& global_params) const;
+    void interact(Person& person, TimePoint t, TimeSpan dt, GlobalInfectionParameters& global_params) const;
```

Comment:
Why isn't this const anymore?

Reply:
The UncertainValue used in some Parameters makes this necessary. Will be changed though due to change of the Parameters.